lib/colorls/flags.rb

```diff
@@ -34,6 +36,52 @@ def process
 
     private
 
+    def add_sort_options(options)
+      options.separator ''
+      options.separator 'sorting options:'
+      options.separator ''
+      options.on('-sMODE', %w[d f], 'sort (f)iles or (d)irectories first') do |m|
+        @opts[:sort] = m == 'd' ? :dirs : :files
+      end
+      options.on('--sort-dirs', '--group-directories-first', 'sort directories first') { @opts[:sort] = :dirs }
+      options.on('--sort-files', 'sort files first')                                   { @opts[:sort] = :files }
+    end
```

Comment:
1. no, optparse only completes the Option itself: `-s`. I'm looking into somehow overriding the behavior of the OptionParser to permit these two options... but I have to read up a bit more on the Ruby syntax and optparse docs.

2. I would like to get rid of these options (see #103), but I would not want to break it in this PR just yet